landoapi/revisions.py

```diff
@@ -34,6 +34,19 @@ def gather_involved_phids(revision: dict) -> set[str]:
     attachments = PhabricatorClient.expect(revision, "attachments")
 
     entities = {PhabricatorClient.expect(revision, "fields", "authorPHID")}
+    revision_diffs = [
+        diff
+        for diff in diffs.values()
+        if PhabricatorClient.expect(revision, "phid")
+        == PhabricatorClient.expect(diff, "fields", "revisionPHID")
```

Comment:
I think we should be only passing the relevant diffs to this method, rather than filtering out irrelevant diffs here.